app/feature/feature-claim-details/src/main/kotlin/com/hedvig/android/feature/claim/details/data/GetClaimDetailUiStateUseCase.kt

```diff
@@ -61,6 +64,14 @@ internal class GetClaimDetailUiStateUseCase(
   private fun ClaimDetailUiState.Content.Companion.fromClaim(claim: ClaimFragment): ClaimDetailUiState.Content {
     val audioUrl = claim.audioUrl
     val memberFreeText = claim.memberFreeText
+
+    val claimType: String? = claim.claimType
+    val incidentDate = claim.incidentDate
+    val submittedAt = claim.submittedAt.toLocalDateTime(TimeZone.currentSystemDefault())
+    val insuranceDisplayName = claim.productVariant?.displayName
+    val termsConditionsUrl =
+      claim.productVariant?.documents?.firstOrNull { it.type == InsuranceDocumentType.TERMS_AND_CONDITIONS }?.url
```

Comment:
They are also taking first, because there is supposed to be only one terms and conditions doc per claim.productVariant. So I think it's fine like this :)